Utilities/File.cpp

```diff
@@ -1898,13 +1900,73 @@ bool fs::file::strict_read_check(u64 _size, u64 type_size) const
 	return true;
 }
 
+std::string fs::get_executable_path()
+{
+#if defined(_WIN32)
+	std::vector<wchar_t> buffer(32767);
+	GetModuleFileNameW(nullptr, buffer.data(), buffer.size());
+	return wchar_to_utf8(buffer.data());
```

Comment:
Because the value of these functions never changes, I suggest doing what fs::get_config_dir does which is wrapping an immediately invoked lambda on the first call of the function into a static const string.